x-pack/plugins/security_solution/server/endpoint/services/actions/action_list.test.ts

```diff
@@ -225,7 +226,7 @@ describe('When using `getActionList()', () => {
         elasticAgentIds: undefined,
         endDate: undefined,
         page: 0,
-        pageSize: undefined,
+        pageSize: 10,
```

Comment:
`page` (above) should never be `0` from the API.

Reply:
a8a0eb693fe3885fbf3089bfe9b90afcda46e75d